projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -566,7 +582,53 @@ public Response getStandings(
 					// we got the internal contest; pass it to the DefaultScoringAlgorithm and get back updated standings
 					try {
 						Properties props = internalContest.getContestInformation().getScoringProperties();
-						String xmlStandings = dsa.getStandings(internalContest, props, logger);
+
+						xlog(logger, "debug 22 scoreboard 2 props "+props);
+						if (props != null) {
+						       Set<Object> set = props.keySet();
+
+						        String[] keys = (String[]) set.toArray(new String[set.size()]);
+
+						        Arrays.sort(keys);
+								xlog(logger, "debug 22   Scoring Properties, there are " + keys.length + " keys");
+
+						        for (String string : keys) {
+									xlog(logger, "debug 22      " + string + "='" + props.get(string) + "'");
+						        }
+						}
+						
+						props = DefaultScoringAlgorithm.getDefaultProperties();
+						xlog(logger, "debug 22  set Default Scoring Props for "+key);
+
+						
+						/**
+						 * Should we use filer by division logic to filter runs ?
+						 */
+						boolean useDivisionFilter = false;
+
+						if (IniFile.isFilePresent()) {
+							new IniFile();
+							useDivisionFilter = StringUtilities.getBooleanValue(IniFile.getValue(SERVER_WTI_BOARD_USE_DIVISIONS_KEY), false);
+						}
+
+						xlog(logger, "useDivisionFilter is " + useDivisionFilter);
+
+						Run[] runs = null;
+						if (useDivisionFilter) {
+							String userLogin= userInformation.getMyClient().getLoginName();
+							xlog(logger, "debug 22 useDivisionFilter userLogin = "+userLogin);
+							Integer clientNumber = StringUtilities.getTeamNumber(userLogin);
+							xlog(logger, "debug 22 useDivisionFilter clientNumber = "+clientNumber);
+							ClientId clientId = new ClientId(internalContest.getSiteNumber(), Type.TEAM, clientNumber);
+							xlog(logger, "debug 22 useDivisionFilter clientId = "+clientId);
+							runs = ScoreboardUtilites.getRunsForUserDivision(clientId, internalContest);
+							
+						} else {
+							runs = internalContest.getRuns();
+						}
+						xlog(logger, "debug 22 useDivisionFilter total runs  = "+runs.length);
```

Comment:
I think outputting this message unconditionally, when the message refers to `useDivisionFilter`, is misleading.  I suggest either dropping the `useDivisionFilter` from the log message, or move the logging up under the "if" condition.

Reply:
It used to be in the useDivisionFilter  block, moved it down.

Reply:
rewrote the log now is:

Reply:
pushed